pwndbg/storage.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import sys
+
+def get_cache_dir():
+    dir = os.environ.get('PWNDBG_CACHE_DIR')
+    if not dir:
+        if sys.platform == 'darwin':
+            dir = os.path.join(os.path.expanduser('~/Library/Caches'), 'pwndbg')
+        elif sys.platform == 'win32':
+            dir = os.path.join(os.getenv('LOCALAPPDATA'), 'pwndbg', 'Cache')
+        else:
+            dir = os.path.join(os.environ.get('XDG_CACHE_HOME') or os.path.expanduser('~/.cache'), 'pwndbg')
+    if sys.version_info[0] >= 3:
```

Comment:
Also if there isn't a better way for `os.makedirs` with `exist_ok=True` kwarg, I would just leave the py2 implementation (as it will work in py3).